CaesarLib/Acts.cs

```diff
@@ -24,5 +25,14 @@ public static String GetAttribute(IWebElement element, String attribute)
         {
             return element.GetAttribute(attribute);
         }
+
+
+        public static void SelectOptionFromDDL(IWebElement element, String value)
```

Comment:
Abbriviation DDL already exists :) It's Definition Data Language. So method name sounds ambiguous